samples/Classic.Demo/MainWindow.axaml

```diff
@@ -300,6 +300,13 @@
           <TextBox Text="Lorem ipsum dolor sit amet.."/>
           <ComboBox ItemsSource="abcdefgh" SelectedIndex="1" />
           <ComboBox ItemsSource="abcdefgh" IsEnabled="False" SelectedIndex="0" />
+          <AutoCompleteBox ItemsSource="{Binding People}" ValueMemberBinding="{ReflectionBinding FirstName}">
```

Comment:
`ValueMemberBinding="{ReflectionBinding FirstName}`
Is not required and can be safely removed, right?

Reply:
No, valuemembinding is required for searching.
I use reflectionbinding because of Rider/plugin bug (cannot recognize element datatype), I will revert to regular binding.